client/state/themes/actions.js

```diff
@@ -407,6 +409,41 @@ export function clearActivated( siteId ) {
 }
 
 /**
+ * Triggers a network request to install theme on Jetpack site.
+ * After installataion it switches page to the customizer
+ * See installTheme doc for install options.
+ * Requires Jetpack 4.4
+ *
+ * @param  {String}   themeId      WP.com Theme ID
+ * @param  {String}   siteId       Jetpack Site ID
+ * @return {Function}              Action thunk
+ */
+export function installAndTryAndCustomize( themeId, siteId ) {
+	return ( dispatch ) => {
+		return dispatch( installTheme( themeId, siteId ) )
+			.then( () => {
+				dispatch( tryAndCustomize( themeId, siteId ) );
+			} );
+	};
+}
+
+export function tryAndCustomize( themeId, siteId ) {
```

Comment:
JSDoc pls? 😄 Sry for nagging, but it's the only function without one in this file, I think...